pytest.ini

```diff
@@ -4,3 +4,4 @@ filterwarnings =
   # Prevent deprecated numpy type aliases from being used
   error:^`np\.[a-z]+` is a deprecated alias for.+:DeprecationWarning:mlflow
   error:^`np\.[a-z]+` is a deprecated alias for.+:DeprecationWarning:tests
+timeout = 60
```

Comment:
```suggestion
timeout = 600
```

60 seconds to test

Reply:
https://github.com/mlflow/mlflow/actions/runs/4827769576/jobs/8600840916?pr=8353#step:6:1286

```
60.00s setup    tests/recipes/regression/v1/test_regression_pipeline_batch_scoring.py::test_recipe_batch_dag_get_artifacts
39.37s call     tests/recipes/test_pipeline.py::test_recipe_get_artifacts
38.08s call     tests/recipes/test_pipeline.py::test_recipes_log_to_expected_mlflow_backend_with_expected_run_tags_once_on_reruns
37.87s call     tests/recipes/test_pipeline.py::test_print_cached_steps_and_running_steps
37.44s call     tests/recipes/regression/v1/test_regression_pipeline.py::test_recipe_run_and_clean_the_whole_recipe_works
37.05s call     tests/recipes/test_pipeline.py::test_recipes_execution_directory_is_managed_as_expected[custom]
36.93s call     tests/recipes/test_cli.py::test_recipes_cli_flow_completes_successfully
36.64s call     tests/recipes/test_pipeline.py::test_test_step_logs_step_cards_as_artifacts
36.00s call     tests/recipes/test_pipeline.py::test_recipes_execution_directory_is_managed_as_expected[None]
34.44s call     tests/recipes/test_cli.py::test_recipes_cli_step_works[register]
=========================== short test summary info ============================
ERROR tests/recipes/regression/v1/test_regression_pipeline_batch_scoring.py::test_recipe_batch_dag_get_artifacts - Failed: Timeout >60.0s
ERROR tests/recipes/regression/v1/test_regression_pipeline_batch_scoring.py::test_recipe_batch_dag_execution_directories - Failed: Timeout >60.0s
ERROR tests/recipes/regression/v1/test_regression_pipeline_batch_scoring.py::test_recipe_batch_dag_clean_step_works[ingest_scoring] - Failed: Timeout >60.0s
ERROR tests/recipes/regression/v1/test_regression_pipeline_batch_scoring.py::test_recipe_batch_dag_clean_step_works[predict] - Failed: Timeout >60.0s
```